components/eam/src/physics/cam/micro_p3.F90

```diff
@@ -830,6 +834,15 @@ SUBROUTINE p3_main_part2(kts, kte, kbot, ktop, kdir, do_predict_nc, do_prescribe
       call ice_water_conservation(qi(k), qidep, qinuc, qiberg, qrcol, qccol, qr2qi_immers_freeze_tend, qc2qi_hetero_freeze_tend, &
            dt, qi2qv_sublim_tend, qi2qr_melt_tend)
 
+      if (Koby_fixes) then
+         call nc_conservation(nc(k), nc_selfcollect_tend, dt, nc_collect_tend, nc2ni_immers_freeze_tend, &
+              nc_accret_tend, nc2nr_autoconv_tend)
+         call nr_conservation(nr(k),ni2nr_melt_tend,nr_ice_shed_tend,ncshdc,nc2nr_autoconv_tend,dt,nr_collect_tend,&
```

Comment:
I think `nc2nr_autoconv_tend` should be replaced with ncautr, since `nc2nr_autoconv_tend` is the tendency of `nc`, while `ncautr` is the tendency of `nr`, at least according to notes in cloud_water_autoconversion. Another fix would be to use `0.5*nc2nr_autoconv_tend` since Delta(nc) = 2*Delta(nr).

Reply:
You're totally right that nc2nr_autoconv_tend is the number of cloud drops turned to rain by autoconversion and is NOT equal to ncautr=the number of rain drops caused by that conversion. ncautr partitions all the new mass into 25 micron radius drops while nc2nr_autoconv_tend removes the fraction of nc proportional to the autoconverted mass divided by the total mass. I'm dubious that autoconversion shouldn't conserve drop number though.

Reply:
Also, we should rename ncautr to nr_autoconv_tend at some point (but not in this PR)

Reply:
@crterai : I don't understand what you mean by "Another fix would be to use 0.5*nc2nr_autoconv_tend since Delta(nc) = 2*Delta(nr)." Are you saying that if I didn't want to use ncautr I could just divide the nc2rn_autoconv_tend value I'm using now by 2? I don't understand where you get this number...

Reply:
@PeterCaldwell As I understand it, autoconversion is the collision of two cloud droplets to form one rain droplet, so that's where the factor of 2 comes in. For every two cloud droplets lost in nc, one nr is gained during autoconversion. But for accuracy, I think it should be replaced with ncautr.

Reply:
Ok, you're right that ncautr is the appropriate variable.